### DISCUSSION TITLE
dhaqsoPay Extension design review

### DISCUSSION THREAD

**Discussion by hayle01** [dugsiiyeinc/dhaqsoPay#31] [dhaqsoPay Extension Developers]
Asxaabey design-kaan ayaan sameeyay oo an ugu tala galay extension-ka qiimeeya waxaan rabnaa inaan bilaabano shaqada team extension developers-ka aan isku raacno design wixii ah maanta ama caawa yay nala dhaafin si aan bari u bilaabano dhisida

kani waa linkiga design-ka
https://www.figma.com/design/NYHcgHxMiTQpBw1w2nsN2h/dhaqsoPay?node-id=0-1&t=SVbllY7GaeIkdZYH-1

**Comment by sharafdin** [comment]
end design-ka cool waye kaliya `quick actions` qeybta sare gee. kadibna aan aragno Insha'Allah!

**Reply by hayle01** [reply]
![image](https://github.com/user-attachments/assets/336f1640-1a6a-465e-a411-f74ef7e25aa9)
eng haddi kor la geeyo quick action sidaan ayuu ku soo baxayaa

**Reply by sharafdin** [reply]
hadeer cool waye eng. dhanka phone number-kana isha ka qaad oo wax micno maleh sido kalena xoogaa dhanka your account padding geli ilaa hoos haso aado Insha'Allah!

**Reply by hayle01** [reply]
Eng lambarka waxaa isha u saaray qof ayaa si live ah ama cashar ahaan video ugu duubi kara haddi uusan rabin in lambarkiisa la arko kawaran.
